tests/Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM postgres:13-alpine
 # Note that this dockerfile needs to be built from the parent directory to
 # be able to copy the initdb.d directory.
 
-RUN apk add go
+RUN apk add go && \
+    apk add gcc && \
+    apk --no-cache --update add build-base
```

Comment:
```suggestion
RUN --no-cache --update add build-base go gcc
```

Reply:
Thanks @jbygdell for your comment. I've merged the code in one line and force pushed.